addons/website_blog/data/website_blog_demo.xml

```diff
@@ -1,222 +1,329 @@
+<figure class="mt-2 mb-4">
+    <img src="https://source.unsplash.com/lRpTgmEsuXc/630x840" class="img-fluid w-100"/>
+    <figcaption class="figure-caption">Photo by PoloX Hernandez, @elpolox</figcaption>
+</figure>
+
+<blockquote class="blockquote my-5">
+    <em class="h4 my-0">Apart from the native population, the local wildlife is also a major crowd puller.</em>
+    <footer class="blockquote-footer">Someone famous in <cite title="Source Title">Source Title</cite></footer>
+</blockquote>
+
+<p>Several migratory and native birds, mammals and reptiles call Copper Canyon their home. The exquisite fauna in this near-pristine land is also worth checking out.</p>
+<figure class="mt-2 mb-4">
+    <img src="https://source.unsplash.com/DPXytK8Z59Y/630x420" class="img-fluid w-100"/>
+    <figcaption class="figure-caption">Photo by Boris Smokrovic, @borisworkshop</figcaption>
+</figure>
+
+<p>A traveler may choose to explore the area by hiking around the canyon or venturing into it. Detailed planning is required for those who wish to venture into the depths of the canyon. There are a number of travel companies that specialize in organizing tours to the region. Visitors can fly to Copper Canyon using a tourist visa, which is valid for 180 days. Travelers can also drive from anywhere in the United States and acquire a visa at the Mexican customs station at the border.</p>
+<p>A holiday to the Copper Canyon promises to be an exciting mix of relaxation, culture, history, wildlife and hiking.</p>
 ]]>
             </field>
         </record>
 
         <record id="blog_post_2" model="blog.post">
-            <field name="name">Integrating your CMS and E-Commerce</field>
-            <field name="subtitle">Building your company's website and selling your products online easy.</field>
+            <field name="name">Maui helicopter tours</field>
+            <field name="subtitle">A great way to discover hidden places</field>
             <field name="blog_id" ref="blog_blog_1"/>
             <field name="is_published" eval="True"/>
             <field name="published_date" eval="time.strftime('%Y-%m-%d %H:%M:%S')"/>
-            <field name="tag_ids" eval="[(6, 0, [ref('blog_tag_1'), ref('blog_tag_2')])]"/>
-            <field name="cover_properties">{"background-image": "url(/web/image/website_blog.blog_post_cover_02)", "resize_class": "cover container-fluid cover_full", "background-color": "oe_black", "opacity": "0.0"}</field>
+            <field name="tag_ids" eval="[(6, 0, [ref('blog_tag_2')])]"/>
+            <field name="cover_properties">{"background-image": "url('https://source.unsplash.com/PB_SdlCPCDs/1000x1000')", "resize_class": "o_wblog_has_cover cover_mid", "text_size_class": "o_wblog_cover_font_big", "background-color": "oe_blue", "opacity": "0.4"}</field>
```

Comment:
Need to replace all unsplash image with local image.
don't worry about the ping for now.

Reply:
It's a pity... I really wanted to reduce the footprint :/
It for people that want to install demo-data offline, right?

ok, I'll add all these pictures 👍